src/components/ProjectInfo.vue

```diff
@@ -20,15 +22,26 @@ export default defineComponent({
       default: false,
     },
   },
-  data: () => ({
+  data: (): {
+    activeTab: TabType | null
+    dialog: boolean
+  } => ({
     activeTab: null,
     dialog: false,
   }),
   computed: {
+    hasTutorialSlot() {
+      return !!this.$slots.tutorial
+    },
     tabs() {
-      const tabs = this.informationPages
-        ? Array.from({ length: this.informationPages.length }, (v, i) => i).concat(['instructions'])
-        : ['instructions']
+      const tabs: TabType[] = [
+        ...Array.from(new Array(this.informationPages?.length ?? 0).keys()),
+        'instructions',
+      ]
+      if (this.hasTutorialSlot) {
```

Comment:
This seems to evaluate to `true`, even when the tutorial is empty. However, we should only push tutorial to tabs, if the tutorial actually has some content, imo.

Background:
Our experimental project types ("Media", "Digitize") are created without tutorials or with an empty string as tutorialId on the project. At the moment, this would result in a tutorial tab without content for these projects.

Reply:
It should be okay for "media" and "digitize" projects as we do not define "tutorial" slot.

Do we need to check if tutorial exists for other project types? Isn't tutorial mandatory for other project types?

Reply:
Media and Digitize projects do not have tutorials at the moment. With a test Media project that had an empty string as value for tutorial, the tutorial tab without content showed up.

However, as Media and Digitize are still experimental and not in production, let's fix this later.